test/parallel/test-process-cpuUsage.js

```diff
@@ -0,0 +1,57 @@
+'use strict';
+require('../common');
+const assert = require('assert');
+
+const result = process.cpuUsage();
+
+// Validate the result of calling with no previous value argument.
+validateResult(result);
+
+// Validate the result of calling with a previous value argument.
+validateResult(process.cpuUsage(result));
+
+// Ensure the results are monotonically increasing.
+let thisUsage;
+let lastUsage = process.cpuUsage();
+for (let i = 0; i < 10; i++) {
+  thisUsage = process.cpuUsage();
+  assert(thisUsage.user >= lastUsage.user);
+  assert(thisUsage.system >= lastUsage.system);
```

Comment:
Sure, why not?  Assuming a fast enough processor that can run this code faster than a microsecond.  In fact, the `cpuUsage()` call **with no argument** (returning cpu usage since process start) could in theory return 0's.

Will ensure tests and comments match that assumption.